jina/types/document/__init__.py

```diff
@@ -1007,14 +1007,13 @@ def convert_image_buffer_to_blob(self, color_axis: int = -1):
         self.blob = to_image_blob(io.BytesIO(self.buffer), color_axis)
 
     def convert_image_blob_to_uri(
-        self, width: int, height: int, resize_method: str = 'BILINEAR'
+        self, resize_method: str = 'BILINEAR', color_axis: int = -1
```

Comment:
Is there a way to keep backward compatibility?

Reply:
will keep `width` and `height`, won't deprecated it (only make this optional)